### PR TITLE
Updated to support Xcode 11.1

### PR DESCRIPTION
**Pull request by reidmain**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: objective-c
-osx_image: xcode11
+osx_image: xcode11.1
 before_script:
-  - gem install cocoapods -v 1.8.0 --no-document
+  - gem install cocoapods -v 1.8.3 --no-document
 script:
-  ./build.sh
+  travis_wait 30 ./build.sh
```
Comment (reidmain) [inline .travis.yml]: Unfortunately Travis CI now seems to take more than 10 minutes to run `pod install` so we need to add this wait to give the tests enough time.

File: FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemo.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -80,7 +81,7 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				0C73704BAABBD042032D31D6 /* BuildFile in Frameworks */,
+				0C73704BAABBD042032D31D6 /* (null) in Frameworks */,
```
Comment (benpious) [inline FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemo.xcodeproj/project.pbxproj]: Huh.

Comment (reidmain) [reply FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemo.xcodeproj/project.pbxproj]: Yeah I know eh? I tried to figure out what the heck was generating this but I couldn't. No setting seemed to map to this.

**Comment by Craystyle1212** [comment]
HTTPS//;

**Pull request merged**